README.md

```diff
@@ -355,18 +369,30 @@ public namespace HungryDevApp
 
             var parameterDict = new Dictionary<string, string>()
             {
-                {"order", "-date_published"},
                 {"page", page.ToString()},
                 {"page_size", pageSize.ToString()}
             };
 
-            PagesResponse<RecipePage> recipePages = butterClient.ListPages<RecipePage>("recipes", parameterDict);
+            PagesResponse<RecipePage> recipePages = butterClient.ListPages<RecipePage>("recipe", parameterDict);
 
-            var viewModel = new RecipeViewModel();
+            var viewModel = new RecipesViewModel();
             viewModel.PreviousPageNumber = recipePages.Meta.PreviousPage;
             viewModel.NextPageNumber = recipePages.Meta.NextPage;
             viewModel.PagesCount = recipePages.Meta.Count;
-            viewModel.RecipePages = recipePages.Data;
+
+            viewModel.Recipes = new List<RecipeViewModel>();
+            foreach (Page<RecipePage> recipe in recipePages.Data)
```

Comment:
Should `recipePages.Data` be lowercase D  :  `recipePages.data`

Reply:
PagesResponse<T>  deserializes "data" and Page<T> deserializes "fields" from the API response to the C# standard Pascal casing. https://github.com/ButterCMS/buttercms-csharp/blob/master/ButterCMS/Models/PagesResponse.cs#L12 and https://github.com/ButterCMS/buttercms-csharp/blob/master/ButterCMS/Models/Page.cs#L8 . If you want these lower case in the classes after deserialization, I can make that change, but it will be a breaking library change.

Reply:
Oh I see - ok. Thanks for the clarification.  Nope this is good thanks.